### DISCUSSION TITLE
Is jinx sending words to suggest possible autocompletions?

### DISCUSSION THREAD

**Discussion by paaguti** [minad/jinx#146] [Q&A] [answered]
Hi,
emacs master as of today, jinx from elpa and company. I've noticed that on LaTeX and org there pop up suggestions of possible completions for a word and I would like to disable them for good. Is jinx responsible for this? If so, is there a way to stop it sending them. They disturb me and slow down Emacs a lot.

Thx, /PA

**Comment by minad** [comment, marked as answer]
Jinx doesn't provide a completion at point function (Capf), so this cannot be caused by Jinx.

**Comment by paaguti** [comment]
Thanks for the answer. Close it then....

On Fri, 8 Mar 2024 at 11:17, Daniel Mendler ***@***.***>
wrote:

> Jinx doesn't provide a completion at point function (Capf), so this cannot
> be caused by Jinx.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/minad/jinx/discussions/146#discussioncomment-8718386>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZKU4GUWVGYYXRMU32AH2DYXGF2NAVCNFSM6AAAAABEJPNNS6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DOMJYGM4DM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Fragen sind nicht da, um beantwortet zu werden,
Fragen sind da um gestellt zu werden
Georg Kreisler

Headaches with a Juju log:
unit-basic-16: 09:17:36 WARNING juju.worker.uniter.operation we should run
a leader-deposed hook here, but we can't yet

**Comment by Thaodan** [comment]
Does Jinx send suggestions to `icomplete-mode` which then uses jinx to complete words?
